addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -330,3 +337,29 @@ test("Mention with @everyone", async () => {
     await click(".o-mail-Composer-suggestion");
     await contains(".o-mail-Composer-input", { value: "@everyone " });
 });
+
+test("@everyone mention adds channel members to the message recipients", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "General",
+        channel_type: "channel",
+        channel_member_ids: [Command.create({ partner_id: serverState.partnerId })],
+    });
+    await start();
+    await openDiscuss(channelId);
+    const messageData = await withUser(serverState.userId, () =>
+        rpc("/mail/message/post", {
```

Comment:
We should avoid doing `rpc` in tests, just send the message from the interface if it is done with the current user.